packages/node-logger/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@digdir/dialogporten-node-logger",
```

Comment:
Tenker du å publisere den på `npm`? Kan den ikke like gjerne være en `workspace`-dependency, eller ser du for at dette er noe flere team skal dra nytte av?

Reply:
Møte med Sean: Skal dobbeltsjekke at noe ikke slikt eksisterer fra før, hvis ikke bør vi slenge det ut slik at andre teams kan bruke det. Bruker workspace for å referere til dette i våre pakker, men publiserer i tillegg.